hoomd/md/pair.py

```diff
@@ -776,7 +782,7 @@ class Morse(_Pair):
         \end{eqnarray*}
 
     See :py:class:`_Pair` for details on how forces are calculated and the available energy shifting and smoothing modes.
-    Use ``coeff.set`` to set potential coefficients.
+    Use ``params`` dictionary to set potential coefficients.
```

Comment:
Should be able to reference this as
```
`params`
```
And `params`  should be documented in an `Attributes:` section, listing the keys to the dictionary and what they are. See the `shape` parameter in `hpmc/integrate.py` for examples of how to document `params`.

`r_cut` and `r_on` in a similar way, but they can be documented in the parent class `_Pair`.